pkgs/development/python-modules/unsloth/default.nix

```diff
@@ -89,6 +91,19 @@ buildPythonPackage rec {
     torchvision
   ];
 
+  propagatedBuildInputs = [ stdenv.cc.cc ];
```

Comment:
A reference in `propagatedBuildInputs` is redundant if you're already linking this path from `$out/$sitePackages/_unsloth_cc.pth`. I can't tell, though, whether it's desirable to bind cc at build time here